app/src/main/java/com/keylesspalace/tusky/db/TimelineDao.kt

```diff
@@ -203,6 +203,9 @@ AND timelineUserId = :accountId
     )
     abstract suspend fun deleteAllFromInstance(accountId: Long, instanceDomain: String)
 
+    @Query("UPDATE TimelineStatusEntity SET filtered = NULL WHERE (serverId = :statusId OR reblogServerId = :statusId)")
+    abstract suspend fun clearWarning(statusId: String): Int
```

Comment:
Needs to also check for the correct accountId as to not accidentally affect statuses from othr accounts with the same id.